client-js/app/elements/labrad-registry.html

```diff
@@ -170,41 +172,75 @@
     }
   </style>
   <template>
-    <div id="reg-area">
-      <!-- Cursor Controls -->
-      <iron-a11y-keys target="[[target]]" keys="up down"
-                      on-keys-pressed="cursorMove"></iron-a11y-keys>
-      <iron-a11y-keys target="[[target]]" keys="right enter"
-                      on-keys-pressed="cursorTraverse"></iron-a11y-keys>
-      <iron-a11y-keys target="[[target]]" keys="left backspace"
-                      on-keys-pressed="cursorBack"></iron-a11y-keys>
-      <!-- Dialog Controls -->
-      <iron-a11y-keys target="[[target]]" keys="esc"
-                      on-keys-pressed="dialogCancel"></iron-a11y-keys>
-      <iron-a11y-keys target="[[target]]" keys="shift+enter"
-                      on-keys-pressed="dialogSubmit"></iron-a11y-keys>
-      <!-- Search Controls -->
-      <iron-a11y-keys target="[[target]]" keys="enter"
-                      on-keys-pressed="searchSubmit"></iron-a11y-keys>
-      <table>
-        <tbody is="selectable-table" id="combinedList" selected="{{selectedIdx}}">
-          <template id="fullList" is="dom-repeat" items="{{listItems}}" filter="filterFunc">
-            <tr name="{{item.name}}"
+  <div id="outer">
+    <!-- Dialog Controls -->
+    <iron-a11y-keys target="[[target]]" keys="esc"
+                    on-keys-pressed="dialogCancel"></iron-a11y-keys>
+    <iron-a11y-keys target="[[target]]" keys="shift+enter enter"
+                    on-keys-pressed="dialogSubmit"></iron-a11y-keys>
+    <!-- Cursor Controls -->
+    <iron-a11y-keys target="[[target]]" keys="up down"
+                    on-keys-pressed="cursorMove"></iron-a11y-keys>
+    <iron-a11y-keys target="[[target]]" keys="right enter"
+                    on-keys-pressed="cursorTraverse"></iron-a11y-keys>
+    <iron-a11y-keys target="[[target]]" keys="left backspace"
+                    on-keys-pressed="cursorBack"></iron-a11y-keys>
+    <!-- Search Controls -->
+    <iron-a11y-keys target="[[target]]" keys="enter"
+                    on-keys-pressed="searchSubmit"></iron-a11y-keys>
+    <paper-header-panel id="left-column">
+      <div class="paper-header" id="buttons">
+        <paper-toolbar>
+          <paper-button on-click="newKeyClicked">
+            <iron-icon icon="create"></iron-icon>
+            new key
+          </paper-button>
+          <paper-button on-click="newFolderClicked">
+            <iron-icon icon="folder-open"></iron-icon>
+            new dir
```

Comment:
When horizontal space gets tight, the "new key" and "new dir" captions render as two lines. We should use `&nbsp;`.